libs/math/include/mrpt/math/gtsam_wrappers.h

```diff
@@ -74,6 +72,46 @@ static gtsam::Matrix6 to_gtsam_se3_cov6(
 	return C;
 }
 
+static void jacobian_rodrigues_from_quat(
```

Comment:
I added a small header.

I'll write the longer expression here in case you want to use it somewhere else :)

The equation we are deriving wrt the quaternion coordinates is:

$\boldsymbol{\omega}= \frac{2 \arccos{q_r}}{|\mathbf{q}_v|}\mathbf{q}_v$

The rows of the resulting Jacobian are:

$\begin{bmatrix} \frac{-(2 q_x)}{((1 - q_r^2)^{1/2}  |\mathbf{q}_v| )} & \frac{(2 acos(q_r) (q_y^2 + q_z^2))}{|\mathbf{q}_v|^3 } & \frac{-(2 q_x q_y acos(q_r))}{|\mathbf{q}_v|^3 } & \frac{-(2 q_x q_z acos(q_r))}{|\mathbf{q}_v|^3 } \\ \end{bmatrix}$
$\begin{bmatrix} \frac{(2 q_y)}{((1 - q_r^2)^{1/2}  |\mathbf{q}_v| )} & \frac{-(2 q_x q_y acos(q_r))}{|\mathbf{q}_v|^3 } & \frac{(2 acos(q_r) (q_x^2 + q_z^2))}{|\mathbf{q}_v|^3 } & \frac{-(2 q_y q_z acos(q_r))}{|\mathbf{q}_v|^3 } \\ \end{bmatrix}$
$\begin{bmatrix}  \frac{-(2 q_z)}{((1 - q_r^2)^{1/2}  |\mathbf{q}_v| )} & \frac{-(2 q_x q_z acos(q_r))}{|\mathbf{q}_v|^3 } & \frac{-(2 q_y q_z acos(q_r))}{|\mathbf{q}_v|^3 } & \frac{(2 acos(q_r) (q_x^2 + q_y^2))}{|\mathbf{q}_v|^3 } \end{bmatrix}$

Reply:
Ah, awesome! Note taken, thanks.